--- a/src/main/patcher.ts
+++ b/src/main/patcher.ts
@@ -147,6 +147,11 @@
             disabledFeatures.add("UseEcoQoSForBackgroundProcess");
             args[1] += [...disabledFeatures].join(",");
         }
+        if (args[0] === "enable-features") {
+            const enabledFeatures = new Set((args[1] ?? "").split(","));
+            enabledFeatures.add("AcceleratedVideoDecodeLinuxGL");
+            args[1] = [...enabledFeatures].join(",");
+        }
         return originalAppend.apply(this, args);
     };
